### PR TITLE
update banner shop and background shop page

### PR DESCRIPTION
**Pull request by Gonaline**

### CODE REVIEW

**Review by FrancoisSalmon — APPROVED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/pages/Shop/index.jsx [hunk 1]
```diff
@@ -57,7 +57,14 @@ export default function Shop() {
...
   return (
     <>
-      <BigPicture img="test" date="" hour="" club1="" club2="" text="" />
+      <BigPicture
+        img="background-boutique"
+        date=""
+        hour=""
+        club1=""
+        club2=""
+        text=""
```
Comment (LoicBrassart) [inline frontend/src/pages/Shop/index.jsx:66]: You could have done better (BgPicture could declare these props as optional)

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**